### PR TITLE
feat/TDP-3108 enforce sharing and commenting

### PR DESCRIPTION
**Pull request by sauravarora04**

### CODE REVIEW

File: packages/article-comments/__tests__/web/shared.web.test.js [hunk 1]
```diff
@@ -117,6 +118,7 @@ it("Render comments label, when comments are loaded", () => {
       onCommentsPress={() => {}}
       commentingConfig={{ account: "sp_pCQgrRiN" }}
       url="dummy-article-url"
+      isCommentEnabledFromZephr
```
Comment (hmgibson23) [inline packages/article-comments/__tests__/web/shared.web.test.js:121]: Can you rename this? As it's not being enabled via Zephr and this is a bit confusing

File: packages/article-magazine-standard/fixtures/full-article.js [hunk 2]
```diff
@@ -2078,7 +2081,10 @@ const makeDefaultConfig = ({
   slug,
   standfirst,
   topics,
-  url
+  url,
+  isSavingEnabledFromZephr,
```
Comment (hmgibson23) [inline packages/article-magazine-standard/fixtures/full-article.js:2085]: Likewise. Can you remove the Zephr as it's a bit confusing?

**Review by hmgibson23 — APPROVED** [review]

File: packages/article-in-depth/fixtures/full-article.js [hunk 3]
```diff
@@ -2082,7 +2082,10 @@ const makeDefaultConfig = ({
   savingEnabled = true,
   textColour = defaultTextColour,
   topics = defaultTopics,
-  url = defaultUrl
+  url = defaultUrl,
+  isSavingEnabled = true,
+  isSharingEnabledFromZephr = true,
```
Comment (othetgyi) [inline packages/article-in-depth/fixtures/full-article.js:2087]: Should we remove the FromZephr bit to make it consistent?

File: packages/article-in-depth/fixtures/full-article.js [hunk 4]
```diff
@@ -2105,7 +2108,10 @@ const makeDefaultConfig = ({
   standfirst,
   textColour,
   topics,
-  url
+  url,
+  isSavingEnabled,
+  isSharingEnabledFromZephr,
```
Comment (othetgyi) [inline packages/article-in-depth/fixtures/full-article.js:2113]: Same comment as above.

File: packages/article-in-depth/fixtures/full-article.js [hunk 5]
```diff
@@ -2287,5 +2293,8 @@ export const testFixture = {
       slug: "topic"
     }
   ],
-  url: "https://url.io"
+  url: "https://url.io",
+  isSavingEnabled: true,
+  isSharingEnabledFromZephr: true,
```
Comment (othetgyi) [inline packages/article-in-depth/fixtures/full-article.js:2298]: Same comment as above.

File: packages/article-magazine-comment/fixtures/full-article.js [hunk 6]
```diff
@@ -1949,7 +1949,10 @@ const makeDefaultConfig = ({
   savingEnabled = true,
   standfirst = defaultStandfirst,
   topics = defaultTopics,
-  url = defaultUrl
+  url = defaultUrl,
+  isSavingEnabled = true,
+  isSharingEnabledFromZephr = true,
```
Comment (othetgyi) [inline packages/article-magazine-comment/fixtures/full-article.js:1954]: Same comment about including Zephr in this name.

File: packages/article-magazine-comment/fixtures/full-article.js [hunk 7]
```diff
@@ -2154,5 +2160,8 @@ export const testFixture = {
       slug: "topic"
     }
   ],
-  url: "https://url.io"
+  url: "https://url.io",
+  isSavingEnabled: true,
+  isSharingEnabledFromZephr: true,
```
Comment (othetgyi) [inline packages/article-magazine-comment/fixtures/full-article.js:2165]: Same comment as above.

File: packages/article-magazine-standard/fixtures/full-article.js [hunk 8]
```diff
@@ -2057,7 +2057,10 @@ const makeDefaultConfig = ({
   sharingEnabled = true,
   standfirst = defaultStandfirst,
   topics = defaultTopics,
-  url = defaultUrl
+  url = defaultUrl,
+  isSavingEnabled = true,
+  isSharingEnabledFromZephr = true,
```
Comment (othetgyi) [inline packages/article-magazine-standard/fixtures/full-article.js:2062]: Same comment about including Zephr in this name.

File: packages/article-magazine-standard/fixtures/full-article.js [hunk 9]
```diff
@@ -2078,7 +2081,10 @@ const makeDefaultConfig = ({
   slug,
   standfirst,
   topics,
-  url
+  url,
+  isSavingEnabled,
+  isSharingEnabledFromZephr,
```
Comment (othetgyi) [inline packages/article-magazine-standard/fixtures/full-article.js:2086]: Same comment as above.

File: packages/article-magazine-standard/fixtures/full-article.js [hunk 10]
```diff
@@ -2271,5 +2277,8 @@ export const testFixture = {
       slug: "topic"
     }
   ],
-  url: "https://url.io"
+  url: "https://url.io",
+  isSavingEnabled: true,
+  isSharingEnabledFromZephr: true,
```
Comment (othetgyi) [inline packages/article-magazine-standard/fixtures/full-article.js:2282]: Same comment as above.

**Review by othetgyi — COMMENTED** [review]
Looks good in general but there are some places where the prop names still have "EnabledByZephr".

**Review by othetgyi — APPROVED** [review]

**Review by gczene — APPROVED** [review]

**Pull request merged**